### PR TITLE
RPM building is back!

### PR DESCRIPTION
**Pull request by mpreisler**
~~Do not merge yet, I want to simplify the spec a little bit, use `make install` and stuff like that.~~

Ready for merging.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -263,6 +341,7 @@ install: dist
 	install -m 0644 README.md $(PREFIX)/$(DOCDIR)/scap-security-guide
 	@# install a symlink in the old content location for compatibility
 	install -d $(PREFIX)/$(DATADIR)/xml/scap/ssg
-	ln -sf $(PREFIX)/$(DATADIR)/scap/ssg $(PREFIX)/$(DATADIR)/xml/scap/ssg/content
+	ln -sf ../../../scap/ssg $(PREFIX)/$(DATADIR)/xml/scap/ssg/content
```
Comment (redhatrises) [inline Makefile]: @mpreisler why not keep `$(PREFIX)/$(DATADIR)`? `../../../` seems like a poor replacement that could be error prone.

Comment (mpreisler) [inline Makefile]: You can't build an RPM in BUILDROOT with that in place. The link destination will be wrong.

Comment (redhatrises) [inline Makefile]: You are right, but are we going to have issues if someone just runs a plain `make install` that is not in an RPM build environment?

**Comment by radzy** [comment]
@redhatrises : yes, we absolutely will have problems with "make install" that is not in an RPM build environment.

**Comment by mpreisler** [comment]
> @redhatrises : yes, we absolutely will have problems with "make install" that is not in an RPM build environment.

Not sure I follow. What exactly is the problem? Can you paste the error you get?

File: Makefile (hunk 1 above)
Comment (redhatrises) [inline Makefile]: I had a really messed up test environment which already had a symlink associated with it, so I was getting errors and infinite link loop. @radzy what @mpreisler did works fine.

**Comment by radzy** [comment]
@mpreisler : Sorry.  I mis-read the code, and I didn't test it.  It's actually backward from what I thought: I'm installing into "DESTDIR=/wr/radzy/wrl8/work/scp/bitbake_build/tmp/work/corei7-64-wrs-linux/scap-security-guide/git-r0/image" , but in the final deployment, it will be put into the normal place "/".  So a symlink to $DESTDIR would be broken, but a relative symlink will work.

**Comment by radzy** [comment]
@redhatrises : In case you didn't see this, or in case my previous comment wasn't clear: I'm OK with this.

**Comment by mpreisler** [comment]
For reference, I proposed to remove rpm building in https://github.com/OpenSCAP/scap-security-guide/pull/1520

File: Makefile [hunk 2]
```diff
@@ -216,8 +245,53 @@ zipfile: dist
...
+	sed -i 's/SSG_MINOR_VERSION.*/SSG_MINOR_VERSION = $(NEW_MINOR_RELEASE)/' $(ROOT_DIR)/VERSION
+	sed -i 's/SSG_RELEASE_DATE.*/SSG_RELEASE_DATE = $(RPM_DATESTR)/' $(ROOT_DIR)/VERSION
+	@echo -e "\nTagging to new release $(NEW_RELEASE)"
+	git add $(RPM_SPEC).in $(ROOT_DIR)/VERSION
+	git commit -m "Make new $(NEW_RELEASE) release"
+	git tag -a -m "Version $(NEW_RELEASE)" v$(NEW_RELEASE)
+
 clean:
-	rm -rf tarball/
+	rm -rf $(RPMBUILD)
```
Comment (redhatrises) [inline Makefile]: @mpreisler in reason as to why `rm -rf tarball` was removed? In fact, shouldn't there be a `rm -rf zipfile` as well?

Comment (mpreisler) [inline Makefile]: Good catch, fixed!

**Comment by redhatrises** [comment]
Builds, installs, and cleans up nicely. Ack.

**Pull request merged**